micro-rdk/src/common/conn/viam.rs

```diff
@@ -558,6 +558,14 @@ where
             None => None,
         };
 
+        if let Some(app) = app_client.as_ref() {
+            if let Ok(device_agent_config) = app.get_agent_config().await {
+                let agent_config: crate::common::config::AgentConfig =
+                    device_agent_config.as_ref().try_into().unwrap();
```

Comment:
will remove the `unwrap` here, but this is mainly used for debugging and demonstrating how I plan to use the new structs/methods in the next PR

Reply:
can we put a TODO remove comment with the next ticket number linked?